packages/core/src/utils/sourceMaps.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getBundleId } from './sourceMaps';
+
+describe('sourceMapUpload utils', () => {
+  beforeAll(() => {
+    delete (global as any).__faroBundleId_foo;
+    delete (global as any).__faroBundleIds;
```

Comment:
I sourceMaps.ts i only see `__faroBundleId_`.
Is the __faroBundleIds list a leftover?